libnfs/Makefile

```diff
@@ -1,12 +1,12 @@
 CFLAGS     = -O2
```

Comment:
Please add a comment to the makefile (syntax is #), and leave info simliar to what was left elsewhere.

Reply:
Done! Thanks.